Cargo.toml

```diff
@@ -32,3 +32,7 @@ rstest = { version = "0.19.0", default-features = false }
 rstest_reuse = "0.7.0"
 tempfile = "3.0"
 xattr = "1.3.1"
+
+[[bin]]
+name = "fuse-ufs-fuser"
```

Comment:
What does the "-fuser" part indicate?  Simply "fuse-ufs" would be more consistent with precedent.

Reply:
There will be a `fuse-ufs-fuse2` for OpenBSD in the future.
The right binary will be choosed when building with `make`.

Reply:
Up until now, `make` hasn't been necessary for building; it's just been a convenience for development.  It would be unfortunate if it suddenly becomes necessary.  Plus, it would not be easy to integrate with things like the ports system.  May I suggest two alternatives?

- Define a `fuser` feature which is on-by-default, but which must be turned off to build on OpenBSD.
- Have a single `src/bin/fuse-ufs` file which is a thin wrapper around either `src/bin/fuse-ufs/fuser.rs` or `src/bin/fuse-ufs/fuse2`, `#[cfg()]`-gated.

Reply:
I like this idea. I'll do this, once I have workspace-ified this project.
I think having `fuser` and `fuse2` features would be useful, with only `fuser` being enabled by default.
On OpenBSD only `fuse2` would work, on all other platforms, you could built with both (if you wanted that for some reason).
It sucks that per-platform dependencies [don't work](https://github.com/rust-lang/cargo/issues/1197)